src/main/resources/io/jenkins/plugins/designlibrary/AppBar/index.jelly

```diff
@@ -1,40 +1,50 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <s:sample>
-    <h2 class="jdl-heading">${%Top app bar}</h2>
-    <div class="jdl-component-sample jdl-component-sample--full-width jdl-component-sample--gridless">
-      <div class="jenkins-app-bar jenkins-!-margin-bottom-0">
-        <div class="jenkins-app-bar__content">
-          <h1>${%title}</h1>
-        </div>
-        <div class="jenkins-app-bar__controls">
+    <p class="jdl-leading-paragraph">${%topAppBar.description}</p>
+    <div class="jdl-component-group">
+      <div class="jdl-component-sample jdl-component-sample--full-width jdl-component-sample--no-bottom-padding jdl-component-sample--gridless">
+        <l:app-bar title="${%title}" subtitle="3">
```

Comment:
So is subtitle always a number? Shouldn't it then be called number or count or something more intuitive? Or is is planned to have there an subtitle text as well? Then it would make sense to have an example for such a page as well.

Reply:
I'm not too sure to be honest - trying not to be overly specific in case there is a future scenario where we might want to include text, only scenario I could currently think of is if we need to provide more context, e.g. https://jenkins-redesign.vercel.app/project/build/375/test-results